src/axolotl/utils/models.py

```diff
@@ -298,10 +305,28 @@ def load_tokenizer(cfg):
         )
     return tokenizer
 
+def load_processor(cfg: DictDefault, tokenizer: PreTrainedTokenizerBase):
+    #model_config = load_model_config(cfg)
+    processor_kwargs = {} #do we actually need this?
+
+    processor_cls = AutoProcessor
+    if cfg.processor_type:
```

Comment:
need to add `processor_type` to https://github.com/axolotl-ai-cloud/axolotl/blob/main/src/axolotl/utils/config/models/input/v0_4_1/__init__.py